### PR TITLE
Upstream updates for py-jupyter-core and py-jupyter-client

### PR DESCRIPTION
**Pull request by pramodk**
Updated required for issues mentioned in https://github.com/jupyter/jupyter/issues/334

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-jupyter-client/package.py [hunk 1]
```diff
@@ -1,17 +1,18 @@
-# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
 # Spack Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack import *
 
-
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-jupyter-client/package.py]: Why? :pineapple:

Two newlines before class definitions…

**Review by matz-e — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-jupyter-client/package.py (hunk 1 above)
Comment (jorblancoa) [reply var/spack/repos/builtin/packages/py-jupyter-client/package.py]: you are right!
`var/spack/repos/builtin/packages/py-jupyter-client/package.py:8: [E302] expected 2 blank lines, found 1`

Comment (pramodk) [reply var/spack/repos/builtin/packages/py-jupyter-client/package.py]: I didn't notice that because I just checkout this package from upstream!

**Comment by matz-e** [comment]
@pramodk, @jorblancoa  updates here?

**Comment by jorblancoa** [comment]
This can be closed, was added to this PR https://github.com/BlueBrain/spack/pull/545/files

**Pull request closed without merging**